app/assets/javascripts/uploadcare/widget/tabs/preview-tab.coffee

```diff
@@ -77,6 +86,24 @@ uploadcare.namespace 'widget.tabs', (ns) ->
             @__setState('image', {file: false})
             @element('image').attr('src', src)
             @initImage(size)
+            df.resolve()
```

Comment:
Should be right after `canvasToBlob` resolve (even if we don't want to show image preview).